### PR TITLE
Java10v2

### PR DESCRIPTION
**Pull request by AustinShalit**

### CODE REVIEW

**Comment by AustinShalit** [comment]
Will try again in the morning to get CI working but this works locally.  @SamCarlberg

**Comment by AustinShalit** [comment]
I disabled all visible testing because of https://github.com/TestFX/TestFX/issues/568

**Comment by SamCarlberg** [comment]
Running `check` locally on my Windows machine causes a JVM crash (https://github.com/javafxports/openjdk-jfx/issues/66).

Appveyor doesn't seem to have ever been configured so it's falsely indicating that everything's OK on win

**Comment by AustinShalit** [comment]
Do you think we should disable all UI Tests on Windows like you did for Shuffleboard?

**Comment by SamCarlberg** [comment]
Yep. Placing a `@DisabledOnOs(OS.WINDOWS)` annotation on every UI test class should do the trick

**Comment by AustinShalit** [comment]
Is it worth adding a custom annotation called UI test that would then disable on windows?

**Comment by SamCarlberg** [comment]
I don't think that's necessary.

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.9-bin.zip
```
Comment (JLLeitschuh) [inline gradle/wrapper/gradle-wrapper.properties:5]: Impressive leap. 👍

**Review by JLLeitschuh — APPROVED** [review]
Lgtm!

**Pull request closed without merging**